src/julia_to_gap.jl

```diff
@@ -219,7 +219,7 @@ function julia_to_gap(
     elseif Wrappers.IsRecord(obj)
         ret_val = NewPrecord(0)
         recursion_dict[obj] = ret_val
-        for x in Vector{String}(Wrappers.RecNames(obj))
+        for x in Wrappers.RecNames(obj)::GapObj
             Wrappers.ASS_REC(ret_val, x, julia_to_gap(Wrappers.ELM_REC(obj, x), recursion_dict; recursive = true))
```

Comment:
(and this code is still wrong, I'll look into fixing it

Reply:
What about
```
xx = Wrappers.RNamObj(x)
Wrappers.ASS_REC(ret_val, xx, julia_to_gap(Wrappers.ELM_REC(obj, xx), recursion_dict; recursive = true))
```

Reply:
Sounds good. I just updated the PR accordingly and added a test.